taniumthreatresponse_connector.py

```diff
@@ -376,21 +376,17 @@ def _get_filename_from_tanium(self, action_result, file_id):
         return RetVal(phantom.APP_SUCCESS, filename)
 
     def _save_temp_file(self, content):
+        vault_tmp_dir = Vault.get_vault_tmp_dir()
```

Comment:
Have you tried with the `add` API?

Reply:
The `vault_add()` method is used [a bit later ](https://github.com/splunk-soar-connectors/taniumthreatresponse/blob/6d2157c16d2b9a9c02b8f9a6e3610db1282ce888/taniumthreatresponse_connector.py#L1180C8-L1180C8). The `_save_temp_file()` is used to extract a content of tanium thread response from a zip file to unpacked file, which is then added to vault.